docs/apis-tools/community-clients/cli-client/index.md

```diff
@@ -30,6 +30,8 @@ export ZEEBE_CLIENT_SECRET='[Client Secret]'
 export ZEEBE_AUTHORIZATION_SERVER_URL='[OAuth API]'
 ```
 
+For Self-Managed environment, the Zeebe API may look like `http://localhost:26500`, or `https://mydomain.com:26500` with OAuth.
```

Comment:
:sweat_smile: Sorry for my comment, I meant in your sample, as it's also using authentication configuration, the typical set up is to use `https` as a scheme, so that's what I was referring to. It's not specific to OAuth per se.

Maybe we can just reword it to "or `https://mydomain.com:26500` when using secure communication."

Reply:
Thank you @npepinpe 😄 Resolved.